sphinx_ncs_theme/static/js/ncs.js

```diff
@@ -19,7 +18,9 @@ function NCS () {
     if (LOCALHOST_RE.test(host)) {
       this.url_prefix = "/";
     } else {
-      this.url_prefix = NCS_PATH_PREFIX;
+      let path = window.location.pathname;
+      let prefix_end_index = path.indexOf("/doc/") + 5;
```

Comment:
Maybe a comment to explain why 5?

Reply:
I thought it implicit due to the "/doc/" string appears just before, but I can add one

Reply:
Fixed, thanks!